src/lib/gadgets/bitwise.unit-test.ts

```diff
@@ -22,6 +22,12 @@ let Bitwise = ZkProgram({
         return Gadgets.xor(a, b, 64);
       },
     },
+    not: {
+      privateInputs: [Field],
+      method(a: Field) {
+        return Gadgets.not(a, 64);
+      },
+    },
```

Comment:
I think we should actually test all values up to the maximum, 255 bit. I didnt do that for XOR, so would be good if you could change that part as well!